tests/test_plugins.py

```diff
@@ -13,3 +14,33 @@ def test_runtime_context_available():
     for plugin in plugins:
         plugin.initialize()
         assert hasattr(plugin, 'runtime_context')
+
+
+@patch('aws_xray_sdk.core.plugins.ec2_plugin.do_request')
+def test_ec2_plugin(mock_do_request):
+    mock_do_request.side_effect = ['token', 'i-0a1d026d92d4709cd', 'us-west-2b',
+                                   Exception("Boom!"), 'i-0a1d026d92d4709ab',
+                                   'us-west-2a',
+                                   Exception("Boom v2!"), Exception("Boom v1!")]
+
+    ec2_plugin = get_plugin_modules(('ec2_plugin',))
+    for plugin in ec2_plugin:
```

Comment:
Is this for loop necessary? It is somewhat confusing since it seems like you're only doing 1 iteration for the `ec2_plugin`

Reply:
Right. Don't really need the loop.